examples/xed-enc-lang.c

```diff
@@ -299,11 +299,11 @@ static void mem_bis_parser_init(mem_bis_parser_t* self, char* s)
             unsigned64_disp = convert_ascii_hex_to_int(self->disp);
             self->disp_width_bits = nibbles*4; // nibbles to bits
             switch (self->disp_width_bits){
-              case 8:  self->disp_val = xed_sign_extend8_64(unsigned64_disp);
+              case 8:  self->disp_val = xed_sign_extend8_64((xed_int8_t)unsigned64_disp);
```

Comment:
I have a separate (massive) patch for a bunch of these sort of issues that I just pushed to my internal repo this week.  Someone was compiling with -Wconversion -Wsign-conversion  and it turned up many of these issues.  So if you give me a couple of days I can push that and hopefully it'll take care of all these issues you've fixed. Sorry about the bad timing.

Reply:
No worries; take whatever you need.